vmpooler.yaml.example

```diff
@@ -161,6 +161,9 @@
 #
 #   - domain
 #     If set, returns a top-level 'domain' JSON key in POST requests
+#
+#   - prefix
+#     If set, prefixes all created VMs with this string.
```

Comment:
Please add `(optional)` to note that this is an optional setting.

Please add a comment indicating this setting should include the separator between `prefix` and the rest of the hostname.